Tests/ReactiveSwiftTests/SchedulerSpec.swift

```diff
@@ -218,6 +218,34 @@ class SchedulerSpec: QuickSpec {
 					scheduler.queue.resume()
 					expect{count}.toEventually(equal(timesToRun))
 				}
+				
+				it("should repeatedly run actions after a given date when the disposable is not retained") {
+					let disposable = SerialDisposable()
+					
+					var count = 0
+					let timesToIncrement = 3
+					
+					// Start two repeating timers, dispose the first, and ensure only the second runs.
```

Comment:
It'd probably be better off as two separate test cases. One for cancellation, while another for not retaining the disposable.